nextjs-13/src/stylesheets/components/_list-newsletter.scss

```diff
@@ -29,6 +29,11 @@
         overflow: hidden;
         text-overflow: ellipsis;
       }
+
+      .icons {
```

Comment:
Avoid plurals: https://nimblehq.co/compass/development/code-conventions/#convention-15 In addition, here it is more about having a list of actions that a user can perform:

```
.list-action {
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/118/commits/b030f0908e4ebbf81c87e3638f4e04d9c1117ff9)